### PR TITLE
[WIP] [Task] Get the image we used

### PR DESCRIPTION
**Pull request by hwchiu**

### CODE REVIEW

File: prepare.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    echo "govendor syncing"
+    govendor sync -v > /tmp/govendor
+    return
+}
+
+dockerimages(){
+    echo "docker images"
+    sudo docker pull prom/prometheus:v2.2.1 > /tmp/dockerimage
+    sudo docker pull hwchiu/ubuntu-nsenter:latest > /tmp/dockerimage
+    sudo docker pull mongo:latest > /tmp/dockerimage
```
Comment (John-Lin) [inline prepare.sh]: image: mongo:4.1-xenial

File: prepare.sh [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    sudo docker pull google/cadvisor:latest > /tmp/dockerimage
+    sudo docker pull gcr.io/kubernetes-helm/tiller:v2.9.1 > /tmp/dockerimage
+    echo "image pull start `date`"
+}
+
+aptget() {
+  echo "apt-get start `date`"
+  sudo apt-get install -y git build-essential openvswitch-switch socat jq httpie
```
Comment (John-Lin) [inline prepare.sh]: sudo apt-get install -qq -y git build-essential openvswitch-switch socat jq httpie

File: prepare.sh [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  sudo add-apt-repository ppa:duggan/bats --yes
+  sudo apt-get update -qq
+  sudo apt-get install -qq bats
```
Comment (John-Lin) [inline prepare.sh]: sudo apt-get install -qq -y bats

**Comment by hwchiu** [comment]
We only cache the govendr/gobuild and it solve the bottleneck for our testing now.

**Pull request closed without merging**